src/components/project/project-rules/project.rules.ts

```diff
@@ -1131,11 +1136,22 @@ export class ProjectRules {
       recipientSession,
     );
 
+    let organization;
+    const perms = this.privileges.for(recipientSession, Organization);
+
+    if (perms.can('read')) {
```

Comment:
This is checking if the recipient can read _all_ organizations, which isn't what we want.
```ts
const orgId = db.query(...);
const org = this.resourcesLoader.load(Organization, orgId);
const name = this.privileges
    .for(recipientSession, Organization, org)
    .can('read', 'name')
  ? org.name : undefined
```